plugins/@grouparoo/pipedrive/src/lib/client.ts

```diff
@@ -0,0 +1,112 @@
+import axios, { AxiosInstance } from "axios";
+
+export class PipedriveClient {
+  request: AxiosInstance;
+
+  constructor(apiToken: string) {
+    this.request = axios.create({
+      baseURL: "https://api.pipedrive.com/v1",
+      params: {
+        api_token: apiToken,
+      },
+    });
+  }
```

Comment:
That's a shame that their API client was unusable (singleton).  That's cool that Axios can automagically append the API Token for us!